src/tx_broadcaster.rs

```diff
@@ -4,7 +4,7 @@ use lightning::chain::chaininterface::BroadcasterInterface;
 use log::error;
 use std::sync::Arc;
 
-pub(crate) struct TxBroadcaster {
```

Comment:
It was required because of `connect_to_peer()` being `pub` as well. But now that I'm not exposing `connect_to_peer()` anymore, I'll revert this change.

Reply:
Fixed.